src/modules/rlm_lua/example.lua

```diff
@@ -1,42 +1,59 @@
 function tprint (tbl, indent)
   if not indent then indent = 0 end
-  for k, v in pairs(tbl) do
+
+  for k, v in tbl.pairs() do
     formatting = string.rep("  ", indent) .. k .. ": "
     if type(v) == "table" then
       print(formatting)
       tprint(v, indent+1)
     else
-      print(formatting .. v)
+      print(formatting .. '"' .. v .. '" (' .. type(v) .. ')')
     end
   end
 end
 
+function preacct()
+  print("example.lua/preacct() Dummy")
+end
 
-function authenticate()
+function accounting()
+  print("example.lua/accounting() Dummy")
+end
 
+function pre_proxy()
+  print("example.lua/pre_proxy() Dummy")
 end
 
-function authorize()
+function post_proxy()
+  print("example.lua/post_proxy() Dummy")
+end
 
+function post_auth()
+  print("example.lua/post_auth() Dummy")
+end
 
-	request['user-name'][2] = nil
+function recv_coa()
+  print("example.lua/recv_coa() Dummy")
+end
 
-        for i in request['user-name'].pairs() do
-                print(i)
-        end
+function send_coa()
+  print("example.lua/send_coa() Dummy")
+end
 
+function detach()
+  print("example.lua/detach() Dummy")
+end
 
-        for k,v in request.pairs_list() do
-                print(k,v)
-        end
+function xlat(arg1, arg2)
+  print("example.lua/xlat() Dummy arg1="..arg1..",arg2="..arg2)
+end
 
+function authenticate()
+  print("example.lua/authenticate() Dummy")
+end
 
-	--tprint(get_attribute("user-name"))
-	--tprint(get_attribute("user-password"))
-	--tprint(get_attribute("tunnel-type", "2"))
-	--print(request['user-name'][0])
-	--print(request['user-name'].next_iter())
-	--print(request['user-name'].next_iter())
-	--tprint(request['user-name'])
-	--tprint(request['user-name'])
```

Comment:
Test is the code that actually tests all the accessors it shouldn't be removed...